ros_google_cloud_language/node_scripts/analyze_text.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
```

Comment:
Do we need shebang while using catkin_virtualenv?? Could you give me advice @knorth55 ...?